meson.build

```diff
@@ -1,8 +1,4 @@
-# frigg is header-only, so we do not need a compiler.
-project('frigg')
-
-# We alreadys need C++ on the build system to build the analyzer.
-add_languages('cpp', native: true)
+project('frigg', 'cpp', default_options: ['cpp_std=c++20'])
```

Comment:
I think the previous dance with `add_languages` was intentional: we need to be able to build frigg when no target system compiler is available.